webExtensionApis/study/api.md

```diff
@@ -196,25 +196,6 @@ Usage scenarios:
   * $ref:
   * optional: false
 
-### `browser.study.surveyUrl( baseUrl, additionalFields )`
```

Comment:
Midpoint survey / url still feels like a real thing.  It can be easily implemented in user land, except that we have conventions for the names of our args.  

+0

Reply:
Yes, browser.study.surveyUrl is still a candidate to add to 5.1+. Just removing it from the schema meanwhile since it has not yet been implemented (like we removed the other things that are yet not implemented).